documentation/modules/exploit/windows/local/cve_2021_40449.md

```diff
@@ -0,0 +1,155 @@
+## Vulnerable Application
+
+A use after free vulnerability exists in the `NtGdiResetDC()` function of Win32k which can be leveraged by
+an attacker to escalate privileges to those of `NT AUTHORITY\SYSTEM`. The flaw exists due to the fact
+that this function calls `hdcOpenDCW()`, which performs a user mode callback. During this callback, attackers
+can call the `NtGdiResetDC()` function again with the same handle as before, which will result in the PDC object
+that is referenced byt his handle being freed. The attacker can then replace the memory referenced by the handle
```

Comment:
byt his -> by this

Reply:
Good spotting, this also affected the module itself so I went ahead and updated it.